static/src/javascripts/bootstraps/enhanced/media-player.js

```diff
@@ -334,59 +278,31 @@ const initPlayButtons = (root: ?HTMLElement): void => {
     });
 };
 
-const initPlayer = (withPreroll: boolean): void => {
-    videojs.plugin('skipAd', skipAd);
+const initPlayer = (): void => {
     videojs.plugin('fullscreener', fullscreener);
 
     fastdom.read(() => {
         $('.js-gu-media--enhance').each(el => {
-            enhanceVideo(el, false, withPreroll);
+            enhanceVideo(el, false);
         });
     });
 };
 
-const initWithRaven = (withPreroll: boolean = false): void => {
+const initWithRaven = (): void => {
     raven.wrap(
         {
             tags: {
                 feature: 'media',
             },
         },
         () => {
-            initPlayer(withPreroll);
+            initPlayer();
         }
     )();
 };
 
 export const initMediaPlayer = (): void => {
-    // The `hasMultipleVideosInPage` flag is temporary until the # will be fixed
-    const shouldPreroll =
-        commercialFeatures.videoPreRolls &&
```

Comment:
Is `commercialFeatures.videoPreRolls` used anywhere else? If not we could remove it from `commercialFeatures`.

Reply:
Thanks for flagging. We are not using it for anything else so I have removed it.